docs/documentation/latex.md

```diff
@@ -30,6 +30,24 @@ f
 ```
 \end{examplefigure}
 
+You can also mix math-mode and text-mode:
+
+\begin{examplefigure}{svg = true}
+```julia
+using CairoMakie
+CairoMakie.activate!() # hide
+
+f = Figure(fontsize = 18)
```

Comment:
I guess the fontsize is superfluous here

Reply:
Yes, I just wanted to see it big! Let me edit